lib/vagrant/action/builtin/cloud_init_wait.rb

```diff
@@ -0,0 +1,22 @@
+module Vagrant
+  module Action
+    module Builtin
+      class CloudInitWait
+
+        def initialize(app, env)
+          @app    = app
+          @logger = Log4r::Logger.new("vagrant::action::builtin::cloudinitwait")
+        end
+
+        def call(env)
+          machine = env[:machine]
+          @logger.info("Waiting for cloud init")
```

Comment:
Maybe this should be moved into the if-statement below just so that it only logs that if we run the wait command?